### PR TITLE
add oneid api server

### PR DESCRIPTION
**Pull request by lixianlin01**
add oneid api server

### CODE REVIEW

File: common-applications/om/om-webserver-oneid/ingress.yaml
Comment (githubliuyang777) [inline common-applications/om/om-webserver-oneid/ingress.yaml:1]: nginx.ingress.kubernetes.io/limit-connections: "20"
nginx.ingress.kubernetes.io/limit-rps: "20"

File: common-applications/om/om-webserver-oneid/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: om-webserver
+  namespace: om-webserver-oneid
+  labels:
+    app: om-webserver
+spec:
+  replicas: 4
```
Comment (githubliuyang777) [inline common-applications/om/om-webserver-oneid/deployment.yaml:9]: replicas: 2

File: common-applications/om/om-webserver-oneid/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+resources:
+- deployment.yaml
+- secrets.yaml
+- ingress.yaml
+- service.yaml
+namespace: om-webserver-oneid
+commonAnnotations:
```
Comment (githubliuyang777) [inline common-applications/om/om-webserver-oneid/kustomization.yaml:7]: commonAnnotations:
  kubernetes.ops.cluster: om-hk-cluster
  kubernetes.ops.email: "jun.zhongjun2@gmail.com"
  kubernetes.ops.os.base: ""
  kubernetes.ops.owner: zhongjun
  kubernetes.ops.region: hk

File: common-applications/om/om-webserver-oneid/secrets.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+kind: SecretDefinition
+metadata:
+  name: om-webserver-secrets
+  namespace: om-webserver-oneid
+spec:
+  name: om-webserver-secrets
+  keysMap:
+    application_properties:
+      path: secrets/data/infra-common/om
+      key: application_properties
```
Comment (zhongjun2) [inline common-applications/om/om-webserver-oneid/secrets.yaml:11]: 修改为当前服务的配置

**Pull request merged**